frontend-react/docs/proposals/0003-uswds-react-components.md

```diff
@@ -0,0 +1,257 @@
+# [0003: Creating New React Components]
+
+* Status: [proposed | rejected | accepted | deprecated | superseded by [0005](0005-example.md)]
```

Comment:
I think we can change this to `accepted` since we're basically sticking with our existing approach, just with a little more formality :)